### ISSUE TITLE
Hours `3-9 PM` interpreted as 3 AM–9 PM

### ISSUE THREAD

**Issue by AndrewKvalheim** [bryceco/GoMap#589]
I ran across a case that the hours recognizer doesn’t handle correctly:

- Input: `3-9 PM`
- Expected output: `15:00-21:00`
- Actual output: `03:00-21:00`

Screenshot:

> <img alt="screenshot" src="https://user-images.githubusercontent.com/1844746/156903883-d6503ff9-93d0-4f06-ac0f-02a2416a0318.jpg" width="375" />

**Comment by bryceco** [comment]
Yeah, that's a pretty annoying edge case. You might think the fix is straightforward except that the code treats a time like "10 PM" as if was "22:00", and "3:00-22:00" makes perfect sense.
